### ISSUE TITLE
Block implicitly retains 'self'

### ISSUE THREAD

**Issue by michael-snapptec** [RNCryptor/RNCryptor-objc#25]
I am getting the above warning message in 6 places in the RNCryptor library. Full message is:

Pods/RNCryptor-objc/RNCryptor/RNDecryptor.m:169:21: 
Block implicitly retains 'self'; explicitly mention 'self' to indicate this is intended behavior

Also found in:
RNDecryptor.m:312:20
RNEncryptor.m:202:23
RNEncryptor.m:213:21
RNEncryptor.m:236:21
RNEncryptor.m:238:20

Everything else with the library seems to be working great! (I'm just trying to get rid of all warnings in my project, whether from my code or not.)

**Comment by NasH5169** [comment]
Missing self-> to explicit the intended behavior... I'm not really understand this warning but... 

```
if (self.hasHMAC) {
      CCHmacUpdate(&_HMACContext, data.bytes, data.length);
    }
```

becomes

```
if (self.hasHMAC) {
      CCHmacUpdate(&self->_HMACContext, data.bytes, data.length);
    }
```

**Comment by rnapier** [comment]
Fixed in #27
